app/src/main/res/layout/view_card_news.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/rtl_container"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <org.wikipedia.feed.view.CardHeaderView
+        android:id="@+id/header_view"
+        android:layout_width="match_parent"
+        android:layout_height="48dp" />
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/news_pager"
+        android:layout_width="match_parent"
+        android:layout_height="288dp" />
+
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/news_item_indicator_view"
+        android:layout_width="wrap_content"
+        android:layout_height="8dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="8dp"
+        android:background="@android:color/transparent"
+        app:tabBackground="@drawable/shape_tab_dot"
+        app:tabGravity="center"
+        app:tabIndicator="@null"
+        app:tabPaddingEnd="8dp"
+        app:tabPaddingStart="8dp"
+        app:tabRippleColor="?attr/color_group_59" />
```

Comment:
The clickable area seems too small and the ripple animation looks slightly weird.

Reply:
I have tried to make it bigger @cooltey but it is still not perfect.. I will discuss it with Robin and make a separate pr to make it better... I am not even sure if he wanted it to be clickable, going by how little space there is between them on the design...

Reply:
For the latest version of the indicators, the spaces between dots are too large. Do you have any updates from Robin?

Reply:
No @cooltey , I am thinking i will ask on Monday after stand-up. Easier on a call.